### DISCUSSION TITLE
绕过中国大陆 IP后，国内网络全断

### DISCUSSION THREAD

**Discussion by wwjj999** [vernesong/OpenClash#3194] [Q&A]
启用绕过中国大陆 IP后，国内网络无法访问，国网都正常。防火墙中ip动态伪装已启用。

**Comment by sinaamos** [comment]
我觉得是你DNS设置错误了吧。CLASH是有DNS解析的，绕过大陆IP的意思是大陆流量不经过CLASH，如果你没有其他DNS服务器，自然就无法解析域名，导致大陆IP的域名无法被解析，无法上网。要么取消勾选，要么在clash下游在搞个dns服务器
